Amigolden/src/app/pages/login/login.page.scss

```diff
@@ -8,4 +8,7 @@ ion-content.login-background{
     --background: none;
     background-image: url('../../../assets/images/login-page.jpg');
     background-size: cover;
+}
+.display-flax{

```

Comment:
I've seen this in a few places, shouldn't we just declare, .display-flax once, and have the scss inherit the class?